Winapp2.ini

```diff
@@ -11866,41 +11866,33 @@ Detect1=HKLM\Software\Big Fish Games\Persistence\Install\F7041T1L1
 Detect2=HKLM\Software\Microsoft\Windows\CurrentVersion\Uninstall\Nancy Drew - Secret of Shadow Ranch - Plus Guide1.0
 DetectFile=%ProgramFiles%\Nancy Drew*
 Default=False
-FileKey1=%ProgramFiles%\Nancy Drew*|dxwebsetup.exe;*.html
-FileKey2=%ProgramFiles%\*\Nancy Drew*|dxwebsetup.exe;*.html
+FileKey1=%ProgramFiles%\*\Nancy Drew*|dxwebsetup.exe;*.html
+FileKey2=%ProgramFiles%\Nancy Drew*|dxwebsetup.exe;*.html
 
-[Natural Selection 2 Cache*]
+[Natural Selection 2*]
```

Comment:
For clarity, it might be better to leave cache in the name, as all this does is just delete the cache files.